plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/TestDeltaLakeMetastoreAccessOperations.java

```diff
@@ -79,6 +80,7 @@ public void testSelect()
         assertMetastoreInvocations("SELECT * FROM test_select_from",
                 ImmutableMultiset.<MetastoreMethod>builder()
                         .add(GET_TABLE)
+                        .add(REPLACE_TABLE)
                         .build());
     }
 
```

Comment:
Let's add as well a statement where the same table is referenced twice (or more) to ensure that `REPLACE_TABLE` is performed at most once.

```
select * from tableA
union
select * from tableA
```